conda/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: scwidgets
+  version: "0.1"
```

Comment:
Extract version from Git, I think via GIT_DESCRIBE_TAG or similar (see how scipp master.yaml does it) https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html
For this to work you will need to create one previous tag.